### ISSUE TITLE
new wifi router setup

### ISSUE THREAD

**Issue by caver456** [ncssar/tech_worklist#14]

**Comment by caver456** [comment]
still need a way to connect to SO wifi - can we just use the SO-provided MiFi that is in the trailer, or personal cell tethering, for the same purpose?

**Comment by robsheldon** [comment]
Not really. Kinda, but part of the point here is to get off of the cellular data usage for things like software updates.
